tailwind.config.js

```diff
@@ -23,6 +23,9 @@ module.exports = {
       fontFamily: {
         sans: ["Figtree", "sans-serif"],
       },
+      screens: {
+        "facility-view": "37rem",
+      },
```

Comment:
let's not do this. "facility-view" is not a "screen". you can use arbitrary values in the inline classes directly instead